README.md

```diff
@@ -62,15 +61,15 @@ variable or `make` parameter), and will soon be moving to v18.
 
 Running with `make EXTRA_ARGS='--language LANG'` will allow you to load `LANG` exclusively, where `LANG` is one for the
 language ids/aliases defined in `lib/languages.ts`. For example, to only run **Compiler Explorer** with C++ support,
-you'd run `make EXTRA_ARGS='--language c++'`. The `Makefile` will automatically install all the third party libraries
-needed to run; using `npm` to install server-side and client side components.
+you'd run `make EXTRA_ARGS='--language c++'`. The `Makefile` will automatically install all the third-party libraries
+needed to run; using `npm` to install server-side and client-side components.
 
 For development, we suggest using `make dev` to enable some useful features, such as automatic reloading on file changes
 and shorter startup times.
 
 You can also use `npm run dev` to run if `make dev` doesn't work on your machine.
 
-Some languages need extra tools to demangle them, e.g. `rust`, `d`, or `haskell`. Such tools are kept separately in the
+Some languages need extra tools to demangle them, e.g. `rust`, `d`, or `Haskell`. Such tools are kept separately in the
```

Comment:
this seems arbitrary, technically all these languages start with a capital letter

Reply:
Could you please fix it? Or should I do it myself?

Reply:
> this seems arbitrary, technically all these languages start with a capital letter

Could you please fix it? Or should I do it myself?

Reply:
Can you please fix it